backend/pkg/sqlite_datasource.go

```diff
@@ -493,3 +493,18 @@ func (datasource *SQLiteDatasource) updateReportContent(id string, reportGroup R
 
 	return reportGroup
 }
+
+// TODO: Handle error cases and also might need to add additional
+// fields i.e. SMTP etc
+func (datasource *SQLiteDatasource) getEmailConfig() EmailConfig {
+	db, _ := sql.Open("sqlite3", datasource.path)
+	defer db.Close()
+
+	var email, password string
+
+	row := db.QueryRow("SELECT email, emailPassword as password FROM Config")
```

Comment:
Very, very tiny nit: 

```
row := db.QueryRow("SELECT email, emailPassword AS password FROM Config")
```

Reply:
Was thinking the SQL queries might need to be in a seperate file?